test/tasks/mock/configs/EnableDeputyPauseModuleTemplate.toml

```diff
@@ -0,0 +1,8 @@
+# this is the file used to determine the network configuration
+
+l2chains = [{name = "OP Testnet", chainId = 11155420}]
```

Comment:
Can we add an example task under `src/improvements/tasks/example/sep`? My PR will pick this up later and simulate it on every CI run.